src/ExportButton.tsx

```diff
@@ -192,6 +191,32 @@ const getTopNVariantsWithinLimit = (
   }
 };
 
+// Extract the properties we support from the initial args so that
+// the query building process can ignore them.
+const getSupportedInitialArgs = (argTypes: ArgTypes, initialArgs: any) => {
+  if (!initialArgs || !argTypes) {
+    return [{}, {}];
+  }
+
+  const supportedArgs = {};
+  const unsupportedArgs = {};
+
+  for (const [argName, arg] of Object.entries(argTypes)) {
+    const argType = getArgType(arg);
+    const initialArgValue = initialArgs[argName];
+
+    if (initialArgValue !== undefined) {
+      if (SUPPORTED_ARG_TYPES.includes(argType)) {
+        supportedArgs[argName] = initialArgValue;
+      } else {
+        unsupportedArgs[argName] = initialArgValue;
+      }
+    }
+  }
+
+  return [supportedArgs, unsupportedArgs];
+};
+
```

Comment:
We need a way to discriminate the initial arguments that we support from the ones we don't, otherwise the diff generated by `buildArgsParam` will always include some `undefined` parameters.
For example, in the button it always included `label: undefined`